validator/distro_validator.go

```diff
@@ -396,6 +396,12 @@ func ensureHasValidHostAllocatorSettings(ctx context.Context, d *distro.Distro,
 			Level:   Error,
 		})
 	}
+	if settings.RoundRule < -1 || 1 < settings.RoundRule {
+		errs = append(errs, ValidationError{
+			Message: fmt.Sprintf("invalid host_allocator_settings.round_rule value of %d for distro '%s' - its value must be -1, 0, or 1", settings.RoundRule, d.Id),
```

Comment:
Should use an enum so the meanings of -1, 0, and 1 aren't hardcoded

Reply:
Not sure if what I did is exactly the pattern you had in mind, but I think it accomplishes the job of demagicking. (I don't want to follow the pattern of GroupVersions because as far as I can tell, you can't unset the bool once you set it in the distro page, and I want to be able to have the default be rounding down, change one distro, and then set the distro back to do whatever the default is, while retaining the option to change the default if it appears advisable.)

Reply:
Ah in that case, I think it's solvable to still use a bool pointer, which is more intuitive for us and the user (although what you have here would work, and does get around the magic number issue). The distro checkboxes could have a third option ("Use Admin Default") or maybe in the UI to unclick radio buttons to denote nil (@john-m-liu would know that).